src/index.js

```diff
@@ -64,13 +64,18 @@ export default class TimeAgo extends Component<DefaultProps, Props, void> {
   timeoutId: ?number;
   isStillMounted: boolean = false;
 
+  // Use Number.isNaN if available, otherwise use polyfill (not available in some browsers inc. IE11)
+  checkIsNaN = (value) => {
+    return (Number.isNaN !== undefined) ? Number.isNaN(value) : typeof value === 'number' && isNaN(value);
+  }
```

Comment:
@chriswilliams-tcc 
I won't recommend using that package as it also add polyfill for ie8 which we don't support.